src/scripts/farming/Plot.ts

```diff
@@ -55,6 +56,20 @@ class Plot implements Saveable {
             return GameConstants.formatTime(timeLeft / growthMultiplier);
         });
 
+        this.formattedTimeLeft = ko.pureComputed(() => {
+            if (this.berry === BerryType.None) {
+                return '';
+            }
+            let timeLeft = 0;
+            if (this.age < this.berryData.growthTime[3]) {
+                timeLeft = Math.ceil(this.berryData.growthTime[3] - this.age);
+            } else {
+                timeLeft = Math.ceil(this.berryData.growthTime[4] - this.age);
+            }
```

Comment:
Are we able to do anything about the magic numbers here (`3`, `4`)?
```suggestion
            if (this.age < this.berryData.growthTime[3]) {
                timeLeft = Math.ceil(this.berryData.growthTime[3] - this.age);
            } else {
                timeLeft = Math.ceil(this.berryData.growthTime[4] - this.age);
            }
```

Reply:
Technically we could use the PlotStage enum, but those are named Bloom/Berry for 3/4, which doesn't really make sense in this context (4 being time to death).

Reply:
Only way I can think of to use PlotStage _(without it becoming confusing)_, is if we added 0 to the `growthTime` in the class:
```js
        this.growthTime = [0, ...this.growthTime];
```

Reply:
I don't want to think right now, so you can refactor it if you want. I've been using magic numbers for this in other locations as well.

Maybe add a getter for ripeTime/deathTime since I only reference 3 and 4 directly in those locations?

Reply:
Yeah, no worries, we can get it done another time, just mentioning it now so it doesn't become too prevalent.